src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/DataSourceUtils.java

```diff
@@ -61,17 +61,21 @@ private DataSourceUtils() {}
     @VisibleForTesting
     static final int MIN_MINOR_VERSION_NUMBER = 6;
     @VisibleForTesting
-    static final int MIN_YEAR_RELEASED        = 2019;
+    static final Calendar MIN_DATE                = new GregorianCalendar(2019, Calendar.JANUARY, 24);
+
+    // Track out maximum version number here:
+    @VisibleForTesting
+    static final int MAX_MAJOR_VERSION_NUMBER = 1;
     @VisibleForTesting
-    static final int MIN_MONTH_RELEASED       = 1;
+    static final int MAX_MINOR_VERSION_NUMBER = 7;
     @VisibleForTesting
-    static final int MIN_DAY_RELEASED         = 24;
+    static final Calendar MAX_DATE            = new GregorianCalendar(2020, Calendar.MAY, 21);
```

Comment:
The `DbSnpIntegrationTest::testDbSnpDataSourceParsing` test does part of this already because it uses the latest version of dbSNP to check that it works.

I can add one in to query the google bucket for the min and max versions though.